src/commands/deploy/command.ts

```diff
@@ -40,12 +46,25 @@ function decideDeployType(): DeployType {
         return DeployType.NextJS;
     }
 
+    // Check if nitro.config.js exists
+    if (
+        fs.existsSync(path.join(cwd, "nitro.config.js")) ||
+        fs.existsSync(path.join(cwd, "nitro.config.mjs")) ||
+        fs.existsSync(path.join(cwd, "nitro.config.cjs")) ||
+        fs.existsSync(path.join(cwd, "nitro.config.ts"))
+    ) {
+        return DeployType.Nitro;
+    }
+
     // Check if "next" package is present in the project dependencies
     if (fs.existsSync(path.join(cwd, "package.json"))) {
         const packageJson = JSON.parse(fs.readFileSync(path.join(cwd, "package.json"), "utf-8"));
         if (packageJson.dependencies?.next) {
             return DeployType.NextJS;
         }
+        if (packageJson.dependencies?.nitro) {
```

Comment:
Looks like the package is named `nitropack` and is placed inside `devDependencies`.
https://github.com/unjs/nitro/blob/v2/examples/middleware/package.json#L9

```suggestion
        if (packageJson.devDependencies?.nitropack) {
```